API_Database.py

```diff
@@ -9,29 +10,31 @@ def insert(date):
     #c.execute("CREATE DATABASE API")
     
     #Create The DB
-    c.execute("CREATE TABLE IF NOT EXISTS API_Results (Date TEXT PRIMARY KEY, Movie_Title TEXT, NASA_URL TEXT, WIKI_URL TEXT)")
+    table_sql = "CREATE TABLE IF NOT EXISTS API_Results (Date TEXT PRIMARY KEY, Movie_Title TEXT, Movie_Url TEXT, Nasa_Title, Nasa_Url TEXT, Wiki_Url TEXT)"
+    c.execute(table_sql)
     
     conn.commit()
 
     '''
     Retrive the data
     '''
-    MOVIE_title = " Movie Title"
-    NASA_data = API_Manager.api_nasa_call_response(date)
-    NASA_data = NASA_data.data
-    NASA_URL = NASA_data['url']
+    search_date = bookmark_data['date']
+    movie_title = bookmark_data['movie_title']
+    movie_url = bookmark_data['movie_img']
+    nasa_title = bookmark_data['nasa_title']
+    nasa_url = bookmark_data['nasa_img']
     
     #API_Manager.api_wiki_call_response(date)
     #WIKI_data = WIKI_data.data
-    WIKI_URL = 'Wiki Page'
+    wiki_url = bookmark_data['wiki_link']
     
     #Insert Data in DB
-    data = (date,MOVIE_title, NASA_URL, WIKI_URL)
+    bookmark_data = (search_date, movie_title, movie_url, nasa_title, nasa_url, wiki_url)
     
     c = conn.cursor()
 
     # Executing the sql Query
-    c.execute("INSERT into API_Results values(?, ?, ?, ?)",data)
+    c.execute("INSERT into API_Results values(?, ?, ?, ?, ?, ?)",bookmark_data)
```

Comment:
Error handling? If your date is a primary key, you'll get an exception if you try to save two rows with the same date